src/instructlab/data/generate.py

```diff
@@ -146,13 +146,13 @@ def generate(
 ):
     """Generates synthetic data to enhance your example data"""
     # pylint: disable=C0415
+    # Third Party
+    from instructlab.sdg.generate_data import generate_data
```

Comment:
is this the final way this will work? How will the user know which sdg version they are running? is that all internal?

For now, I feel like that is fine if we just do the work for them but eventually I feel like this should be surfaced

Reply:
the code will certainly evolve. The goal at this iteration was moving the CLI code as close to as-is as possible